spec/library_spec.rb

```diff
@@ -0,0 +1,51 @@
+require './lib/library.rb'
+require './lib/person.rb'
+describe Library do
+  person = Person.new("Carissa")
```

Comment:
When you need to define a variable to use withing your specs make use of `let`

e.g
```ruby
let(:person) { Person.new('Carrissa') }
```
More on that [here](http://www.betterspecs.org/#let)